elements/power.lua

```diff
@@ -258,7 +276,10 @@ local Enable = function(self, unit)
 		self:RegisterEvent('UNIT_FACTION', Path)
 
 		if(power:IsObjectType'StatusBar' and not power:GetStatusBarTexture()) then
-			power:SetStatusBarTexture[[Interface\TargetingFrame\UI-StatusBar]]
+			if(power.colorPower and power.statusbar and power.atlasEnable) then
+				statusbar = power.statusbar
```

Comment:
The layout should not be required to add extra information for internal logic, a better way of handling this is to set the original statusbar variable after the fallback code, and just use normal methods of getting it.

Basically, after [this line](https://github.com/oUF-wow/oUF/blob/441ee4be0d1140f7861e416609760cf7003a36be/elements/power.lua#L262) you could do statusbar = power:GetStatusBarTexture() (but you also have to consider the posibility that the object might not be a statusbar, requiring a teensy bit more logic).

Also, different statusbars might have different textures, storing them globally for the entire element is not a good idea, you should store it on the element itself (e.g. power._originalTexture).